.gitignore

```diff
@@ -1,3 +1,5 @@
+data/*
```

Comment:
You may already know, but Git won't commit an empty folder. You could put an empty `.gitignore` file inside `data/` if you want to keep the folder but ignore the files it contains. (Sometimes people call this empty file `.gitkeep`, since the file is just there to tell Git to keep the folder.)